src/Exports/TranslationsExport.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Brackets\AdminTranslations\Exports;
+
+use Maatwebsite\Excel\Concerns\FromCollection;
+use Maatwebsite\Excel\Concerns\WithMapping;
+use Maatwebsite\Excel\Concerns\WithHeadings;
+use Brackets\AdminTranslations\Translation;
+use Illuminate\Support\Facades\Auth;
+use Brackets\Translatable\Facades\Translatable;
+
+class TranslationsExport implements FromCollection, WithMapping, WithHeadings
+{
+
+    private $exportLanguage;
+
+    public function __construct($request)
+    {
+        $this->exportLanguage = $request->exportLanguage;
+        $this->templateLanguage = $request->templateLanguage;
+    }
+    /**
+     * @return \Illuminate\Support\Collection
+     */
+    public function collection()
+    {
+        return Translation::all();
+    }
+
+    public function headings(): array
+    {
+        $headings = [
+            trans('brackets/admin-translations::admin.fields.namespace'),
+            trans('brackets/admin-translations::admin.fields.group'),
+            trans('brackets/admin-translations::admin.fields.default'),
+            mb_strtoupper($this->exportLanguage),
```

Comment:
Please order the columns in that order, that the language column is last, so if there is a reference column, it should be before column for translation.